x-pack/plugins/security/public/index.ts

```diff
@@ -15,7 +15,6 @@ import {
 } from './plugin';
 
 export { SecurityPluginSetup, SecurityPluginStart };
-export { SessionInfo } from './types';
```

Comment:
__note:__ it doesn't seem like we want it to be a part of our public API, feels like our own internal thing.